lib/delta.ex

```diff
@@ -578,20 +582,29 @@ defmodule Delta do
     base = push(base_rest, base_remaining)
     other = push(other_rest, other_remaining)
 
-    case {base_op, other_op} do
-      {%{"insert" => str}, %{"insert" => str}} ->
-        attrs = Delta.Attr.diff(base_op["attributes"], other_op["attributes"])
-        delta = push(delta, Op.retain(op_len, attrs))
-
-        do_diff(base, other, diffs, delta, :equal, len - op_len)
-
-      _ ->
-        delta =
-          delta
-          |> push(Op.delete(op_len))
-          |> push(other_op)
+    delta =
+      case {base_op, other_op} do
+        {%{"insert" => str}, %{"insert" => str}} ->
```

Comment:
I would name this "ins" instead of "str" because str can be an object. The key requirement here is just that the two are equal values. Actually it might be a good idea to use `cond` to make this explicit but we do end up with an extra unreachable branch so could go either way:

```elixir
    delta =
      cond do
        base_op["insert"] == other_op["insert"] ->
          # ...
        is_map(base_op["insert"]) ->
          # 
        true ->
          raise("Unreachable")
      end
```

Reply:
yeah, calling it "str" is leftover from my previous PR adding diff, but you're right, this path is also for when it's two equal objects.